### DISCUSSION TITLE
Is www.heycolleagues.com built on Angular 18 with SSR feature ?

### DISCUSSION THREAD

**Discussion by Kanhaiya2909** [community/community#132830] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Is [Hey Colleagues](https://www.heycolleagues.com/Entertainment/Human-Interest-Stories/heycolleagues-by-keynate/1b48fb50-932b-48bb-a72b-da1014969d91) built using Angular 18 with server-side rendering (SSR)? It was developed by [Keynate](https://www.keynate.com/).

I'm impressed with the performance of this Angular application, which scores above 60. Check it out: [PageSpeed Insights](https://pagespeed.web.dev/analysis/https-www-heycolleagues-com/eh38a7j5wv?form_factor=desktop).

**Comment by ryantusi** [comment, marked as answer]
Yes, it is built on Angular 18 with SSR. Score above 60 is a good one but not great, the developers might to follow measures for performance for touching 90 to be a great one. I
